pkgs/by-name/xi/xivlauncher/package.nix

```diff
@@ -25,7 +41,7 @@ in
     nugetDeps = ./deps.nix; # File generated with `nix-build -A xivlauncher.passthru.fetch-deps`
 
     dotnetFlags = [
-      "-p:BuildHash=${rev}"
+      "-p:BuildHash=${source.rev}"
```

Comment:
Should this be `rev` or `version`? They're the same with upstream XIVLauncher, but RB's fork differentiates them